Storage/SQL/BrowserSchema.swift

```diff
@@ -949,508 +676,25 @@ open class BrowserSchema: Schema {
-
-            // We don't specify a title, expecting it to be generated on the fly, because we're smarter than Android.
-            // We also don't migrate the 'id' column; we'll generate new ones that won't conflict with our roots.
-            let migrateArgs: Args = [BookmarkRoots.MobileFolderGUID]
-            let migrateLocal = """
-                INSERT INTO bookmarksLocal (guid, type, bmkUri, title, faviconID, local_modified, sync_status, parentid, parentName)
-                SELECT guid, type, url AS bmkUri, title, faviconID, \(modified) AS local_modified, \(status) AS sync_status, ?, ''
-                FROM bookmarks WHERE type IS \(BookmarkNodeType.bookmark.rawValue)
-                """
-
-            // Create structure for our migrated bookmarks.
-            // In order to get contiguous positions (idx), we first insert everything we just migrated under
-            // Mobile Bookmarks into a temporary table, then use rowid as our idx.
-
-            let temporaryTable = """
-                CREATE TEMPORARY TABLE children AS
-                SELECT guid
-                FROM bookmarks
-                WHERE type IS \(BookmarkNodeType.bookmark.rawValue)
-                ORDER BY id ASC
-                """
-
-            let createStructure = """
-                INSERT INTO bookmarksLocalStructure (parent, child, idx)
-                SELECT ? AS parent, guid AS child, (rowid - 1) AS idx FROM children
-                """
-
-            let migrate: [(String, Args?)] = [
-                (migrateLocal, migrateArgs),
-                (temporaryTable, nil),
-                (createStructure, migrateArgs),
-
-                // Drop the temporary table.
-                ("DROP TABLE children", nil),
-
-                // Drop the old bookmarks table.
-                ("DROP TABLE bookmarks", nil),
-
-                // Create indices for each structure table.
-                (indexBufferStructureParentIdx, nil),
-                (indexLocalStructureParentIdx, nil),
-                (indexMirrorStructureParentIdx, nil),
-            ]
-
-            if !self.run(db, queries: prep) ||
-               !self.prepopulateRootFolders(db) ||
-               !self.run(db, queries: migrate) {
-                return false
-            }
-            // TODO: trigger a sync?
-        }
-
-        // Add views for the overlays.
-        if from < 14 && to >= 14 {
-            let indexMirrorStructureChild = """
-                CREATE INDEX IF NOT EXISTS idx_bookmarksMirrorStructure_child
-                ON bookmarksMirrorStructure (child)
-                """
-
-            if !self.run(db, queries: [
-                self.bufferBookmarksStructureView,
-                self.localBookmarksStructureView,
-                indexMirrorStructureChild]) {
-                return false
-            }
-        }
-
-        if from == 14 && to >= 15 {
-            // We screwed up some of the views. Recreate them.
+        if from <= 42 {
```

Comment:
So we are on version 42, and we'll drop the view we deleted in this PR right?

Reply:
Oh, I made a mistake with the logic here. I don't want this to happen just yet until all of the file is cleaned up so I'll set it to == 42 for now and put it back when it's done.